README.md

```diff
@@ -1,583 +1,27 @@
-# gemma.cpp
+---
+library_name: gemma.cpp
```

Comment:
A model card can be useful, but let's not overwrite the previous README.md with it :) Please revert this file, we can add another file if you like.